packages/yourloops/components/dashboard-widgets/medical-files/prescription-list.tsx

```diff
@@ -87,6 +87,14 @@ export default function PrescriptionList({ teamId, patientId }: CategoryProps):
             className={`${classes.hoveredItem} ${prescription.id === hoveredItem ? "selected" : ""}`}
             onMouseOver={() => setHoveredItem(prescription.id)}
             onMouseOut={() => setHoveredItem(undefined)}
+            onClick={() => MedicalFilesApi.getPrescription(patientId, teamId, prescription.id).then((data) => {
```

Comment:
Can you move it into an async/await function on top, for a better readablity

Reply:
moved in a function but keeping .then (unless you find it better to have empty promise ...)